crowbar_framework/app/helpers/barclamp/suse_manager_client_helper.rb

```diff
@@ -25,7 +25,7 @@ def suse_manager_client_rpm
     end
 
     def suse_manager_client_install
-      "/opt/dell/bin/barclamp_install.rb --rpm suse-manager-client"
+      "knife cookbook upload core -o /opt/dell/chef/cookbooks"
```

Comment:
would be good to use a crowbar lib like https://github.com/crowbar/crowbar-core/blob/master/crowbar_framework/lib/crowbar/chef/upload.rb, but it seems it cannot upload single cookbooks